### DISCUSSION TITLE
Scrobble/watch not working

### DISCUSSION THREAD

**Discussion by beeeeswax** [Taxel/PlexTraktSync#796] [Support] [answered]
I've installed PTS on a Windows 10 machine using the zip download, Plex Media Server is running on the same machine.

Synchronizing watched status is working fine but when I start PTS with the watch switch, it seems to be listening correctly but it doesn't seem to like any of the events and doesn't scrobble anything.  When shows/movies are played and hit the watched threshold, all I see is this:

> Listening for events!
INFO: Starting AlertListener: wss://192-168-0-3.xxxxxxxxxxxxxxxxxxx.plex.direct:32400/:/websockets/notifications?X-Plex-Token=<hidden>
ERROR: AlertListener Msg Error: 'Tag' object has no attribute 'ratingKey'
ERROR: AlertListener Msg Error: 'Tag' object has no attribute 'ratingKey'
ERROR: AlertListener Msg Error: 'Tag' object has no attribute 'ratingKey'
ERROR: AlertListener Msg Error: 'Tag' object has no attribute 'ratingKey'

My Info output:

> PlexTraktSync Version: 0.18.x
Python Version: 3.10.2 (tags/v3.10.2:a58ebcc, Jan 17 2022, 14:12:15) [MSC v.1929 64 bit (AMD64)]
Plex API Version: 4.9.1
Trakt API Version: 3.4.0
Cache Dir: C:\Plex\PlexTraktSync
Config Dir: C:\Plex\PlexTraktSync
Log Dir: C:\Plex\PlexTraktSync
Plex username: xxxxxxxxxxxxxxx
Trakt username: xxxxxxxxxxx
Plex Server version: 1.25.5.5492-12f6b8c83, updated at: 2022-02-09 13:11:49
Enabled 2 libraries in Plex Server: ['Movies', 'TV']

**Comment by glensc** [comment, marked as answer]
bugs need to be reported as issues!

**Reply by glensc** [reply]
https://github.com/Taxel/PlexTraktSync/commit/861f7437c9783985f5443bf77cd3b4d01e4bc095
